src/tests/krate/publish.rs

```diff
@@ -144,24 +151,30 @@ fn new_krate() {
 
 #[test]
 fn new_krate_with_token() {
-    let (_, _, _, token) = TestApp::full().with_token();
+    let (app, _, _, token) = TestApp::full().with_token();
 
     let crate_to_publish = PublishBuilder::new("foo_new").version("1.0.0");
     let json: GoodCrate = token.publish_crate(crate_to_publish).good();
 
     assert_eq!(json.krate.name, "foo_new");
     assert_eq!(json.krate.max_version, "1.0.0");
+
+    let files = app.stored_files();
+    assert_eq!(files, vec!["crates/foo_new/foo_new-1.0.0.crate"]);
```

Comment:
This feels like the kind of thing we might end up doing a fair bit, so I wonder if we might want to add a helper to `TestApp` that can take a slice of paths and return `true` if they're the only stored files — at which point we can just wrap that in an `assert!` within the test.

(Personal bias alert: I tend to like small test helpers like that if they help express what the test is actually testing in their name, since it cuts through the boilerplate.)

Reply:
IMHO the assertion has little boilerplate so I'm not sure a dedicated assertion helper fn is necessary, but I'm open to suggestions.

due to your statement above I'll consider this a non-blocking concern that can be addressed after merging :)

Reply:
> due to your statement above I'll consider this a non-blocking concern that can be addressed after merging :)

Definitely!